### ISSUE TITLE
Ease of Use: Chat Context modal

### ISSUE THREAD

**Issue by DacoDev** [sourcegraph/app#49]
From my personal experience, the chat context modal where you can select your repo is a little confusing. I can't seem to find where to add additional repos.

When I first loaded Cody, I added one of my repos and that's fine. But if I want to add another, the only place to do it seems to be in the chat context menu, and inside there, I wish that I could either have tab-completed file path traversal or a system file picker pop-up.

I tried entering the absolute path to another one of my repos but it didn't seem to do anything. It also seems like I was only able to open the Advanced Settings page for a while, which doesn't include the ability to add a repo, until for some reason a normal settings page opened and it had the add repo options. 

tl;dr please consider adding an easier way to add a local/remote repo to the chat context modal since that seems like where the app wants you to add a repo

Thanks!

edit: after getting the second repo added to Cody, I now see that "Add repositories..." refers specifically to adding another codebase to the current chat's context; so maybe that could more clearly say something like "Select repositories..." or longer "Select repositories for chat context..." etc; and maybe a little + button on the right side of that text box tool-tipped "Add a repo to Cody" or some such function. Thanks again!
